### ISSUE TITLE
MoNuSeg Dataset

### ISSUE THREAD

**Issue by 3D-ICE-arXiv** [3D-ICE-arXiv/Papers#116]
The dataset consists of diverse H&E stained tissue
images captured from seven different organs (e.g., breast, liver, kidney, prostate,
bladder, colon and stomach), which were collected from 18 institutes. The dataset
has a public training set and a public test set for network training and evaluation,
respectively. The training set contains 30 histopathological images with
hand-annotated nuclei, while the test set consists of 14 images. The size of the
histopathological images is 1000 × 1000 pixels

https://monuseg.grand-challenge.org/Data/

Naylor, P., Lae, M., Reyal, F., Walter, T.: Segmentation of nuclei in histopathology
images by deep regression of the distance map. IEEE Trans. Med. Imaging 38(2),
448–459 (2018)

**Comment by 3D-ICE-arXiv** [comment]
https://github.com/3D-ICE-arXiv/Papers/issues/84
